strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/User.java

```diff
@@ -26,6 +30,7 @@
 
     private String securityTokenKey;
 
+    @Embedded
```

Comment:
Shouldn't `AccessModel` be `Embeddable` in this case ? When do we support this annotation in our codebase ?

Reply:
OrientDB support this

Reply:
> Shouldn't AccessModel be Embeddable in this case ? 

oh, now I see what you mean, it's `javax.persistence.Embeddable` annotation. 
But it's seems that OrientDB works without this with just `javax.persistence.Embedded`.
So as I sayed it's supports this as it is.

Reply:
ok, seems that OrientDB should work even without `javax.persistence.Embedded`, and I have a feeling that it assume that type is embedded if the field is not annotated wtih `ManyToOne` or other relation type.

Reply:
So we can get rid of `javax.persistence.Embedded` from our source code so there won't be any misunderstandings to use it or not ?

Reply:
the thing is that `Embedded` types have no `rid`. they just embedded as a part of enclosing entity. in general it's just ORM sugar.

Reply:
I think it can be useful just as info that this types are just Embedded.